config.js.example

```diff
@@ -22,5 +22,5 @@ global.ALLOWED_CLIENTS = [
   // W3C:
   /https?:\/\/(www\.)?w3c?\.org/i,
   // GitHub:
-  /https?:\/\/github\.com/i
+  /https?:\/\/.*\.github\.io/i
```

Comment:
This would let `http://malicious.com/github.io/boom` pass.
I suggest `/https?:\/\/[^.]+\.github\.io/i`